oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticSpellcheckTest.java

```diff
@@ -1,238 +0,0 @@
-/*
```

Comment:
Why was this class removed ?

Reply:
This class contains 3 tests:
* testSpellcheckSingleWord
* testSpellcheckSingleWordWithDescendantNode
* testSpellcheckMultipleWords

The same tests are present in `SpellCheckCommonTest`with slightly different names (no `testSpellcheck` prefix):
* singleWord
* singleWordWithDescendantNode
* multipleWords

Reply:
ah. ok